src/resources/images/moon.svg

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
The folder name should be `assets` sorry for mentioning `resources` in the previous review, that convention is for android projects.